Makefile

```
@@ -154,3 +154,15 @@ restart: clean
 .PHONY: frestart
 frestart: fclean
 	@make --silent start
+
+
+snos:
```

```suggestion
.PHONY: snos
snos:
```

This is important so that `Make` does not get confused between `snos` as a file and a build target.